code/modules/admin/topic.dm

```diff
@@ -11,9 +11,29 @@
 		return TRUE
 	log_admin_private("[key_name(usr)] clicked an href with [msg] authorization key! [href]")
 
-/datum/admins/Topic(href, href_list)
-	..()
+// BASE ADMINS_TOPIC
+
+/datum/admins_topic()
+	var/keyword
+	var/log = TRUE
+	var/key_valid

```

Comment:
Since this is unnecessary with the admin_topics I would remove this. Probably replace it with a list that contains the href_list.

Reply:
href_list goes through the tryRun via input.

Reply:
Ah I see.